blockchain/types/Cargo.toml

```
@@ -7,19 +7,24 @@ edition = "2021"
 [dependencies]
 # MobileCoin dependencies
 mc-account-keys = { path = "../../account-keys" }
+mc-attest-core = { path = "../../attest/core"}
 mc-common = { path = "../../common", default-features = false }
 mc-crypto-digestible = { path = "../../crypto/digestible", features = ["dalek", "derive"] }
 mc-crypto-keys = { path = "../../crypto/keys", default-features = false }
 mc-crypto-ring-signature = { path = "../../crypto/ring-signature", default-features = false }
 mc-transaction-core = { path = "../../transaction/core" }
 mc-transaction-types = { path = "../../transaction/types" }
 mc-util-from-random = { path = "../../util/from-random" }
+mc-util-serial = { path = "../../util/serial"}
 
 # External dependencies
 displaydoc = { version = "0.2", default-features = false }
 hex_fmt = "0.3"
+hex = "0.4.3"
 prost = { version = "0.10", default-features = false, features = ["prost-derive"] }
 serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
+toml = "0.5.9"
+url = "2.2.2"
```

Optional/Equivalent: Omit the patch version here, for fewer dependabot edits to this file.

```suggestion
hex = "0.4"
prost = { version = "0.10", default-features = false, features = ["prost-derive"] }
serde = { version = "1.0", default-features = false, features = ["alloc", "derive"] }
toml = "0.5"
url = "2.2"
```